### PR TITLE
Getting started documentation

### LINKED ISSUE #124: Documentation: "Getting Started" page
Create a getting started page with all the steps to:
- [x] setup the dependencies
- [ ] show an avatar from an email address
- [x] fetch and show a profile

### PR DESCRIPTION
**Pull request by maxme**
Closes #124

### Description

This is the first draft of the "Getting started" documentation. We briefly describe how to create an API Key and provide instructions for quickly integrating and using the SDK.

**Note:** I'll merge the PR when the backend issues are clarified. (And #161 is also merged)

### Testing Steps

- No testing is needed, just documentation review.

### CODE REVIEW

File: docs/get-started/get-started.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+
+# Get Started with Gravatar-SDK-Android
+
+## Setup
```
Comment (mlumeau) [inline docs/get-started/get-started.md]: I think this header would work better.

```suggestion
## Creating an API key
```

File: docs/get-started/get-started.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+You'll need an API key to use the SDK's full capabilities. You can use some features without an API Key, but you'll receive limited information, so we highly encourage you to create one in the [developers' portal](https://gravatar.com/developers/).
```
Comment (mlumeau) [inline docs/get-started/get-started.md]:
```suggestion
You'll need an API key to use the SDK's full capabilities. You can use some features without an API Key, but you'll receive limited information, and stricter rate limits may apply, so we highly encourage you to create one in the [developers' portal](https://gravatar.com/developers/). 
```

File: docs/get-started/get-started.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+}
+```
+
+Then you can access the API key in your app's code like this:
+
+```kotlin
+Gravatar.initialize(BuildConfig.GRAVATAR_API_KEY)
+```
+
+## Usage
```
Comment (mlumeau) [inline docs/get-started/get-started.md:75]: Could we add a section on how to use the API methods to fetch a profile without displaying it in a UI component?

**Review by mlumeau — CHANGES_REQUESTED** [review]
Great simple guide,  straight to the point. I suggested some changes related to the API.

File: docs/get-started/get-started.md (hunk 3 above)
Comment (hamorillo) [reply docs/get-started/get-started.md:75]: Done! Great suggestion!

**Comment by hamorillo** [comment]
Hey @mlumeau, do you think we can merge this PR?

**Comment by mlumeau** [comment]
As part of the 1.0.0 release, yes.

**Review by mlumeau — APPROVED** [review]

**Pull request merged**